DEVELOPMENT.md

```diff
@@ -33,7 +33,7 @@
 
     ```sh
     mvn test          # unit tests
-    bazel test //...  # integration tests
```

Comment:
I took care of this file in https://github.com/googleapis/gapic-generator-java/pull/1351